### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by boxed**
I was hoping to use this lib to grab arrow keys, but that doesn't seem to work on OSX (I just get ESC back), but here are some other fixes :(

I tried to make it work but I don't know much about the intricacies of terminal escape codes and whatnot so I didn't find out what was wrong. Grabbing two characters in a row for example seems to get me ESC and then it blocks on the next character even if I hit an arrow key...

### CODE REVIEW

**Comment by magmax** [comment]
Thank you for you contribution.

Please, could you merge again? I've changed some of the lines you modified (in setup.py, I removed the "pep" and "flake8" requirements)

**Comment by boxed** [comment]
Should be good to go now I hope

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pep8     == 1.5.7
+flakes   >= 0.1.9
+doublex  >= 1.8.1
+pexpect  >= 3.3
+coverage >=3.7.1,<4.0a1
+
+pytest     >= 2.6.2
+pytest-cov >= 1.8.0
+
+wheel >= 0.24.0
```
Comment (magmax) [inline requirements-dev.txt]: Do you really require this file?

All this should be automatically installed with:

```
    python setup.py test
    python setup.py flake8
```

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+--index-url https://pypi.python.org/simple/
+
+-e .
```
Comment (magmax) [inline requirements.txt]: Same for this file.

File: README.rst [hunk 3]
```diff
@@ -99,13 +99,13 @@ In order to develop or running the tests, you can do:
...
 
-   . venv/bin/activate
+   source venv/bin/activate
 
 4. Install dependencies
 
 .. code:: bash
 
-    pip -r requirements.txt -r requirements-dev.txt
+    pip install -r requirements.txt -r requirements-dev.txt
```
Comment (magmax) [inline README.rst]: Now this shouldn't be necessary.

**Comment by magmax** [comment]
I'm going to merge it, because my comments are not relevant enough. But I'd like to know your oppinion about them.

Thank you!

File: README.rst (hunk 3 above)
Comment (boxed) [inline README.rst]:
```
(venv)➜ python-readchar git:(master) ✗ pip -rrequirements.txt

Usage:   
  pip <command> [options]

no such option: -r
(venv)➜ python-readchar git:(master) ✗ pip --version
pip 1.5.6 from /[..]/python-readchar/venv/lib/python2.7/site-packages (python 2.7)
```

I don't know what version of pip you're using, but it should absolutely be necessary as far as I can tell

File: requirements.txt (hunk 2 above)
Comment (boxed) [inline requirements.txt]: Again.. the README says to use that file.

File: requirements-dev.txt (hunk 1 above)
Comment (boxed) [inline requirements-dev.txt]: Your README says "ip -r requirements.txt -r requirements-dev.txt". I was just trying to make that actually work. I guess you could remove that part from the README?

File: README.rst (hunk 3 above)
Comment (boxed) [inline README.rst]: Oh wait, you mean "this entire command should no longer be necessary"?

Yea, ok then just remove it :P

File: requirements-dev.txt (hunk 1 above)
Comment (magmax) [inline requirements-dev.txt]: I'm trying to do so, but I wanted to know your opinion before that.

This is possible thanks to the latest changes. It was required before.

File: README.rst (hunk 3 above)
Comment (magmax) [inline README.rst]: Ok then. Maybe next change. I will double test it.

File: requirements-dev.txt (hunk 1 above)
Comment (boxed) [inline requirements-dev.txt]: Ok, yea I agree.

**Pull request merged**